doc/tutorials/enable_ivshmem.rst

```diff
@@ -63,10 +39,37 @@ steps:
         communication and separate it with ``:``. For example, the
         communication between VM0 and VM2, it can be written as ``0:2``
 
-   .. note:: You can define up to eight ``ivshmem`` hv-land shared regions.
-
 - Build with the XML configuration, refer to :ref:`getting-started-building`.
 
+Ivshmem DM-Land Usage
+*********************
+
+Follow ``Enable Ivshmem Support`` and
+and add this line as an ``acrn-dm`` boot parameter::
+
+     -s slot,ivshmem,shm_name,shm_size
+
+where
+
+-  ``-s slot``  - Specify the virtual PCI slot number
+
+-  ``ivshmem``  - Virtual PCI device name
```

Comment:
This phrasing makes me believe, as a user, that I may have to pick a name and use it there. So how about something like this instead:
```suggestion
-  ``ivshmem``  - Virtual PCI device emulating the Shared Memory
```

Reply:
make sense, will update.